### PR TITLE
expires: 0 works now

### PR DESCRIPTION
**Pull request by asyncanup**

### CODE REVIEW

**Comment by jhusain** [comment]
LGTM.

File: lib/support/incrementVersion.js [hunk 1]
```diff
@@ -1,4 +1,8 @@
 var version = 1;
 module.exports = function incrementVersion() {
+    // console.log(new Error().stack);
```
Comment (steveorsomethin) [inline lib/support/incrementVersion.js]: Minor: lingering comment

Comment (asyncanup) [reply lib/support/incrementVersion.js]: thanks for spotting!

Comment (steveorsomethin) [reply lib/support/incrementVersion.js]: Otherwise LGTM. Feel free to self-merge.

**Pull request merged**